hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/DFSTestUtil.java

```diff
@@ -206,6 +206,8 @@
   private static final String[] dirNames = {
     "zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine"
   };
+
+  private static final int WAIT_REPLICATION_ATTEMPTS = 40;
```

Comment:
Why you have pulled this up? Means nobody apart from `waitReplication ` method is using it? We could have kept it there itself, the variable is private, no one can change it also

Reply:
Thanks @ayushtkn for the feedback.
I removed that variable in the latest push.

I was thinking to get all hardcoded variables in one place so that it will be easier to clean-up and refactor code, but anyway, this is not necessary for this jira